Adyen/settings.py

```diff
@@ -4,8 +4,10 @@
 ENDPOINT_CHECKOUT_TEST = "https://checkout-test.adyen.com"
 ENDPOINT_CHECKOUT_LIVE_SUFFIX = "https://{}-checkout-live" \
                                 ".adyenpayments.com/checkout"
-API_CHECKOUT_VERSION = "v40"
+API_CHECKOUT_VERSION = "v41"
 API_CHECKOUT_UTILITY_VERSION = "v1"
 API_RECURRING_VERSION = "v25"
-API_PAYMENT_VERSION = "v40"
+API_PAYMENT_VERSION = "v41"
 API_PAYOUT_VERSION = "v30"
+LIB_VERSION = "1.4.0"
```

Comment:
It is currently already 1.4.0. Shouldn't this be 1.4.1?

Reply:
Breaking change in v41 so 2.0.0